libraries/Microsoft.Bot.Builder.LanguageGeneration/Evaluator.cs

```diff
@@ -287,7 +287,7 @@ public object ConstructScope(string inputTemplateName, List<object> args)
             }
 
             var parameters = TemplateMap[templateName].Parameters;
-            var currentScope = _evaluationTargetStack.Count > 0 ? CurrentTarget().Scope : new CustomizedMemory(null);
+            var currentScope = CurrentTarget().Scope;
```

Comment:
Why change this one.

Reply:
This is the original code. In last PR for https://github.com/microsoft/botbuilder-dotnet/pull/3802, I changed this line to make sure constructScope can work in injecting LG, where _evaluationTargetStack.Count is 0.
In current solution, we don't need constructScope  any more. So I modified the code to the original.